railties/lib/rails/generators/test_unit/scaffold/scaffold_generator.rb

```diff
@@ -30,17 +34,21 @@ def fixture_name
 
       private
 
-        def attributes_hash
-          return if attributes_names.empty?
+      def attributes_string
+        attributes_hash.map {|k, v| "#{k}: #{v}"}.join(", ")
+      end
 
-          attributes_names.map do |name|
-            if %w(password password_confirmation).include?(name) && attributes.any?(&:password_digest?)
-              "#{name}: 'secret'"
-            else
-              "#{name}: @#{singular_table_name}.#{name}"
-            end
-          end.sort.join(", ")
-        end
+      def attributes_hash
+        return {} if attributes_names.empty?
+
+        attributes_names.map do |name|
+          if %w(password password_confirmation).include?(name) && attributes.any?(&:password_digest?)
+            ["#{name}", "'secret'"]
+          else
+            ["#{name}", "@#{singular_table_name}.#{name}"]
+          end
+        end.sort.to_h
+      end
```

Comment:
The rubocop build is failing for these because the methods should be indented 2 spaces under `private` and there should be a space after the bracket here `{|k, v| "#{k}: #{v}"}.join(", ")`.